modules/developers-guide/pages/webpack-config.adoc

```diff
@@ -5,46 +5,101 @@ The tutorials introduced a few basic approaches to building a front-end user int
-  const outputRoot = path.join(__dirname, dfxJson.defaults.build.output, name);
-  const filename = path.basename(value.main, ".mo");
-  return {
-    ...acc,
-    ["ic:canisters/" + name]: path.join(outputRoot, filename + ".js"),
-    ["ic:idl/" + name]: path.join(outputRoot, filename + ".did.js"),
-  };
-}, {});
+const aliases = Object.entries(dfxJson.canisters).reduce(
+  (acc, [name, _value]) => {
+    // Get the network name, or `local` by default.
+    const networkName = process.env["DFX_NETWORK"] || "local";
+    const outputRoot = path.join(
+      __dirname,
+      ".dfx",
+      networkName,
+      "canisters",
+      name
+    );
+
+    return {
+      ...acc,
+      ["dfx-generated/" + name]: path.join(outputRoot, name + ".js"),
+    };
+  },
+  {}
+);
 ----
 
 === Entry and output configuration
 
-After creating canister aliases, the template `+webpack.config.js+` file generates a webpack configuration that adds the canister aliases automatically as imported modules in the default front-end entry point defined in the project's `+dfx.json+` file.
-For example, using the default settings an import similar to the following is added to the `+src/CANISTER_assets/public/index.js+` file:
+After creating canister aliases, the template `+webpack.config.js+` file generates a webpack configuration for the front-end and adds the resulting files to the output directory for your project directory. 
+
+In many cases, you can use the default `+webpack.config.js+` file as-is, without any modification, or you can add plug-ins, modules, and other custom configuration to suit your needs.
+The specific changes you make to the `+webpack.config.js+` configuration largely depend on the other tools and frameworks you want to use.
+
+For example, if you have experimented with the link:tutorials/custom-frontend{outfilesuffix}[Customize the front-end] or link:tutorials/my-contacts{outfilesuffix}[Add a stylesheet] front-end tutorials, you might have modified the following section to work with React JavaScript:
 
 [source,js]
 ----
-import CANISTER-ALIAS from 'ic:canisters/CANISTER-ALIAS';
+    // Depending in the language or framework you are using for
+    // front-end development, add module loaders to the default
+    // webpack configuration. For example, if you are using React
+    // modules and CSS as described in the "Adding a stylesheet"
+    // tutorial, uncomment the following lines:
+    // module: {
+    //  rules: [
+    //    { test: /\.(ts|tsx|jsx)$/, loader: "ts-loader" },
+    //    { test: /\.css$/, use: ['style-loader','css-loader'] }
+    //  ]
+    // },
+    plugins: [
+      new HtmlWebpackPlugin({
+        template: path.join(__dirname, info.frontend.entrypoint),
+        filename: 'index.html',
+        chunks: ['index'],
+      }),
+      new webpack.ProvidePlugin({
+        Buffer: [require.resolve('buffer/'), 'Buffer'],
+        process: require.resolve('process/browser'),
+      }),
+    ],
+  };
+}
```

Comment:
Some of the code highlighted isn't necessarily relevant to its related segue. Might be better to trim down the example to show only the following:

```suggestion
    module: {
      rules: [
        { test: /\.(ts|tsx|jsx)$/, loader: "ts-loader" },
        { test: /\.css$/, use: ['style-loader','css-loader'] }
      ]
    },
    plugins: [
      new HtmlWebpackPlugin({
        template: path.join(__dirname, info.frontend.entrypoint),
        filename: 'index.html',
        chunks: ['index'],
      })
    ],
  };
}
```